src/main/scala/higherkindness/rules_scala/common/sbt-testing/JUnitXmlReporter.scala

```diff
@@ -8,8 +8,11 @@ import scala.collection.mutable.ListBuffer
 import scala.xml.{Elem, XML}
 
 class JUnitXmlReporter(tasksAndEvents: ListBuffer[(String, ListBuffer[Event])]) {
-  private def escape(str: String): String =
-    str.replaceAll("&", "&amp;").replaceAll("\"", "&quot;").replaceAll("<", "&lt;").replaceAll(">", "&gt;");
+  private def escape(info: Any): String = info match {
```

Comment:
Why did this become an `Any`?

Reply:
I think it was to capture nulls; Scala doesn't have union types

Reply:
Since it's possible to take null type here, using String type also works.